server/pom.xml

```diff
@@ -2335,6 +2335,21 @@
       </properties>
     </profile>
 
+    <!-- Deploy to sonatype for release -->
+    <profile>
+      <id>release</id>
+      <distributionManagement>
+        <snapshotRepository>
+          <id>sonatype-nexus-snapshots</id>
+          <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+        </snapshotRepository>
+        <repository>
+          <id>sonatype-nexus-staging</id>
+          <url>https://oss.sonatype.org/service/local/staging/deploy/maven2/</url>
```

Comment:
@definite So the plan is to deploy server **release** artifacts to the sonatype **snapshots** repo? That seems weird.

Reply:
The behavor is the same with other artfacts.

Reply:
So until now, zanata-parent was configured to use cloudbees for snapshots and releases, and this was inherited by most of the zanata modules. However, this was overridden in server to use cloudbees instead.

My understanding of this change: when we use the `release` profile (during releases, I presume), we will deploy server artifacts to sonatype instead of cloudbees. 

@definite This sort of change **really** needs a JIRA issue.

But assuming we now think it's okay to deploy server artifacts to sonatype for releases, surely snapshots would be fine too? In fact, why don't we just get rid of the cloudbees configuration and inherit the sonatype config from zanata-parent? There seems to be no need to deal with Maven profiles at all.